app/upload/views.py

```diff
@@ -72,3 +82,22 @@ def upload_document(service_id):
             'mimetype': mimetype,
         }
     ), 201
+
+
+def _upload_document_with_hashed_recipient_email(verification_email, service_id, file_data, mimetype):
+    hasher = PasswordHasher(
+        memory_cost=15360,
+        time_cost=2,
+        parallelism=1,
+        hash_len=16,
```

Comment:
Where did we get this from? :) I don't recognise it. Is it just the default that the library uses and we're surfacing here to be explicit?

Reply:
Yeah, I took it from here:

https://argon2-cffi.readthedocs.io/en/stable/parameters.html

Let me know if you think we should not pass it explicitly.

Reply:
It's fine - thanks for clarifying :)